QRCoder/QRCoder.csproj

```diff
@@ -1,12 +1,13 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net35;net40;netstandard1.1;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net35;net40;netstandard1.1;netstandard2.0;net5.0-windows</TargetFrameworks>
```

Comment:
Yes, you are using WPF and WinForms features. This forces your code to be windows only.
I can give it a try to separate it to **net5.0** only in case there is code available.

Reply:
But QRCoder.csproj (!= QRCoderDemo, etc.) isn't Windows only. As it compiles to netstandard2.0 and runs fine on Linux, I wouldn't say it's Windows only. ;-)

Then let's better change to both...
```suggestion
    <TargetFrameworks>net35;net40;netstandard1.1;netstandard2.0;net5.0;net5.0-windows</TargetFrameworks>
```
...and add some directives like "#NET5" and "#NET5_WIN" to include/exclude parts from compilation that work only with net5.0-windows.